packages/server/src/server.ts

```diff
@@ -43,12 +43,21 @@ const cors = require('koa-cors');
 const app = new Koa();
 const rpc = koaJsonRpc();
 
+const responseSuccessLabel = 'Success';
+const responseFailureLabel = 'Failure';
 const register = new Registry();
 collectDefaultMetrics({ register, prefix: 'rpc_relay_' });
 const methodSuccessLatencyCounter = new Counter({
```

Comment:
Just `methodSuccessCounter` since there is now `methodLatencyHistogram`?

Reply:
A histogram should already include a timer along with it, so can probably just drop the counter. If that's not the case then the counter should be converted to a timer as counters for requests are not useful, IMO.

Reply:
Took both your suggestions and simplified